webapp/components/Modal/ReportModal.vue

```diff
@@ -40,10 +62,55 @@ export default {
     id: { type: String, required: true },
   },
   data() {
+    let valuesReasonCategoryOptions = [
+      'discrimination_etc',
+      'pornographic_content_links',
+      'glorific_trivia_of_cruel_inhuman_acts',
+      'doxing',
+      'intentional_intimidation_stalking_persecution',
+      'advert_products_services_commercial',
+      'criminal_behavior_violation_german_law',
+      'other',
+    ]
+
+    let reasonCategoryOptions = []
+    valuesReasonCategoryOptions.forEach(reasonCategory => {
+      reasonCategoryOptions.push({
+        label: this.$t('report.reason.category.options.' + reasonCategory),
+        value: reasonCategory,
+      })
+    })
+
     return {
       isOpen: true,
       success: false,
       loading: false,
+      failsValidations: true,
+      form: {
+        reasonCategory: null,
+        reasonCategoryOptions,
+        reasonDescription: '',
+      },
+      formSchema: {
```

Comment:
done... test added... although this feels a bit overboard because the user cannot choose something that is not a predetermined options, so it must be one of the enums we provide as radio buttons
Also, we are not using the `errors`, so the disabled button is based on the `form.reasonCategory` not being `falsy`, so this is kinda acting like our `formSchema`